modules/uploading-model-files-to-pvc.adoc

```diff
@@ -0,0 +1,53 @@
+:_module-type: PROCEDURE
+
+[id="uploading-model-files-to-pvc_{context}"]
+
+= Uploading model files to a Persistent Volume Claim (PVC)
+
+When deploying a model, you can serve it from a preexisting Persistent Volume Claim (PVC) where your model files are stored. You can upload your local model files to a PVC by using a running workbench.
+
+.Prerequisites
+
+* You have access to the {productname-short} dashboard.
+* You have a running code-server or Jupyter workbench.
```

Comment:
Suggest: changing:
You have a running code-server or Jupyter workbench.
to:
You have access to a project that has a running workbench.